### DISCUSSION TITLE
Word export not opening

### DISCUSSION THREAD

**Discussion by LauraGOP** [monarc-project/MonarcAppFO#471] [General]
Hello,

When I download the context validation report, I get this message when I try to open the word document. I tried with another risk assessment and I didn't have the issue. Have you ever had any issue like this?

Kr,
![Screenshot 2022-11-24 091501](https://user-images.githubusercontent.com/101174751/203730525-b2ae3ef3-76db-4e00-b454-f8090bb31146.png)

**Comment by ruslanbaidan** [comment]
Hello,

We can recall the same issue happened with another report on your side (you asked before).
Somehow the custom reports that were uploaded in your system (on our cloud or your cloud) are write protected or in a broken format.

You could ask the person who uploaded the custom report to check this and regenerate as a correct MS Word document.

Best regards

**Reply by LauraGOP** [reply]
Hello, the reports are the not customized. They're the ones provided in the instance.

Thanks

**Reply by ruslanbaidan** [reply]
Hello,

Could you please tell where is the Monarc instance hosted ?
Did you try to open the document in another application (not MS Word) ?
Based on the error message on your screenshot this is most luckily related to permissions on the file.

**Reply by LauraGOP** [reply]
The instance is located on premise.
I tried to open it with another app but it does not work. 
What I find strange is that I tried opening the same document of another risk assessment and it worked perfectly. I opened the template and it also worked.

**Reply by ruslanbaidan** [reply]
Unfortunately we can't determine the issue without having an access to the analysis.
What you could do is to create a new analysis based on this one, remove all the essential data from it (Context modelling are required in the way how they are now), export it and send the file to us by email.
So we can check on our side.

**Reply by LauraGOP** [reply]
Ok, thanks. Should I send it at info@cases.lu?

**Reply by ruslanbaidan** [reply]
Yes, on this address please

**Reply by jfrocha** [reply]
We found and [fixed](https://github.com/monarc-project/zm-client/commit/193e2e396e595862b08e20b586c843a65e7d14bf) an issue in the 3rd deliverable generation not related to the 2nd.

The issue was a risk operation scale label with special character. "Health & Safety"

Please check by removing the ampersand character (&) if the issue is resolved for you.
